docs/resources/users.md.erb

```diff
@@ -58,6 +58,20 @@ The following examples show how to use this Chef InSpec audit resource.
       it { should exist }
     end
 
+### Test only allowed users exist
+
+    allowed_users = %w(user1 user2 user3)
+
+    users.where { uid > 1000 && uid < 65534 }.usernames.sort.each do |u|
```

Comment:
No need for the upper bound I suspect.

Reply:
Maybe not, in our environment it's the `nfsnobody` user but happy to omit it for the example.